Jellyfin.Plugin.AniList/Providers/AniList/ApiModel.cs

```diff
@@ -162,10 +162,22 @@ public float GetRating()
         /// <returns></returns>
         public List<string> GetStudioNames()
         {
+            PluginConfiguration config = Plugin.Instance.Configuration;
+
             List<string> results = new List<string>();
             foreach (Studio node in this.studios.nodes)
             {
-                results.Add(node.name);
+                if (config.OnlyAnimationStudios)
+                {
+                    if (node.isAnimationStudio)
+                    {
+                        results.Add(node.name);
+                    }
+                }
+                else
+                {
+                    results.Add(node.name);
+                }
```

Comment:
```suggestion
                if ((!config.OnlyAnimationStudios || node.isAnimationStudio) && !results.Contains(node.name))
                {
                    results.Add(node.name);
                }
```

I'd like to suggest this instead, that should also get rid of duplicates as mentioned here https://github.com/jellyfin/jellyfin-plugin-anilist/pull/81#issuecomment-2585192452

Aside from this, this seems to be working well.